### PR TITLE
"About my self and work" section in homepage

### PR DESCRIPTION
**Pull request by AxelSoler**
Following template #4
added "about my self section"
created work section
used grid CSS and Flex to personalize the webpage
updated README file

### CODE REVIEW

**Review by JuliCarracedo — CHANGES_REQUESTED** [review]
Hi @AxelSoler , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [ ] You are not following any of the given [Figma designs](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) in your application. This is a requirement since you must be able to parse a UI design into a product. Kindly modify your design to fit any of the 5 figma templates.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AxelSoler** [comment]
hello @JuliCarracedo, thank you for your answer but I am using template 4. 
It is designed for a mobile website as it is in the requirement
Captura de pantalla 2022-03-17 190303
Captura de pantalla 2022-03-17 190156
Captura de pantalla 2022-03-17 190134

**Comment by AxelSoler** [comment]
hello @JuliCarracedo, did you receive my comment?

File: style.css [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+  src: url(./Font/Roboto/Roboto-Black.ttf);
+}
+
+@font-face {
+  font-family: Inter;
+  src: url(./Font/Roboto/Roboto-Light.ttf);
+}
+
 body {
   background-color: rgb(7, 7, 88);
```
[the remark this answers is not in the document]
Comment (AxelSoler) [reply style.css:17]: hello @chaw-bot , thank you for the feedback but I still don't understand, I think am using the same background image

File: README.md [hunk 2]
```diff
@@ -2,7 +2,11 @@
...
 My second Mycroverse proyect: beggining of my portfolio
 
-page designed for mobiles
+Page designed for mobiles based on template 4 of the given Figma designs
+
+Updated:
+I am using CSS grid and Flex in the web page 
+the is more description in the home page (title story, picture, description and button) 
 
 Built With html and CSS
```
[the remark this answers is not in the document]
Comment (AxelSoler) [reply README.md]: hello @chaw-bot , thank you for the feedback but I am using the README template provided by Mycroverse

**Comment by AxelSoler** [comment]
hello @chaw-bot , thank you for the feedback
Can I keep the "about page" section? I think it looks better than the Figma template
And don't you see the title for the project page?
image
Also, I don't understand what are you talking wit those marks, I am looking at the same thing as in the Figma template
image
I really appreciate all your responses and feedback
thank you

**Comment by AxelSoler** [comment]
hello @chaw-bot, please can you check the pull request and tell me if it is ok? I only have one chance to send a review
Thank you very much

**Review by MeqdamAlqudah — APPROVED** [review]
# Hi @AxelSoler , 

### Status: Approved ✔️

:+1:  You have implemented all the suggested changes made by the previous reviewer. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**